external/storm-hdfs/src/main/java/org/apache/storm/hdfs/bolt/HdfsBolt.java

```diff
@@ -80,6 +86,11 @@ public HdfsBolt addRotationAction(RotationAction action){
         return this;
     }
 
+    public HdfsBolt withTickTupleIntervalSeconds(int interval) {
```

Comment:
Could give a more meaningful name to convey the actual usage (e.g `withFlushIntervalSeconds`). I think we need to have a default value for this to ensure a sync is `always` done even if user doesn't specify this option and also ensure the option value is within some lower and upper thresholds so that tuples are acked within TOPOLOGY_MESSAGE_TIMEOUT_SECS and a sync doesn't happen too frequently.